main.rb

```diff
@@ -0,0 +1,145 @@
+# rubocop:disable Metrics/MethodLength
+# rubocop:disable Metrics/CyclomaticComplexity
+
```

Comment:
- [ ] It would be nice to NOT disable cops to ensure consistency with best practices. Disabling the cops can easily lead to anti-patterns and bad practices.

- [ ] Linter should be set up correctly for this repository, but there's no `rubocop.yml` file found in this repository. It would be great to set it up and make sure all linters are passing. You can find the linters configuration here [linters](https://github.com/microverseinc/linters-config/tree/master/ruby)